### DISCUSSION TITLE
Cant add more columns through prisma

### DISCUSSION THREAD

**Discussion by Figure26** [prisma/prisma#12931] [Q&A]
I cant for some reason add another variablethrough the prisma.create function. I have 7 columns im adding already when im creating a row but when i add the 8 it just keeps getting errors. Ive double checked everything and cant for the life of me see whats wrong.

The API:
```ts
export default async function (req: NextApiRequest, res: NextApiResponse) {
  const prisma = new PrismaClient({ log: ["query"] });

  try {
    const {userman, date, title, kommentar, starttid, sluttid, colorsy, dateint} = req.body;
    const result = await prisma.kalender.create({
      data: {accountid: userman, datum: date, titel: title, kommentar: kommentar, starttid: starttid, sluttid: sluttid, viktigt: colorsy, datumint:dateint,
      },
    });

    res.status(201);
    res.json(result );
  } catch (e) {
    console.error(e);

    res.status(500);
    res.json({ error: "Sorry unable to save bokning to database" });
  } finally {
    await prisma.$disconnect();
  }
}
```

The function from index.tsx

```ts
const submitData = async (e) => {
const body = {
        userman,
        date,
        title,
        kommentar,
        starttid,
        sluttid,
        colorsy,
        dateint
      };
      await fetch("../../api/create1", {
        method: "POST",
        headers: { "Content-Type": "application/json" },
        body: JSON.stringify(body),
      });
};
```

The schema

```prisma
model kalender {
  id        Int      @id @default(autoincrement())
  accountid String?
  titel     String?
  kommentar String?
  viktigt   String?
  datum     DateTime
  starttid  String?
  sluttid   String?
  theColor  String?
  start     DateTime @default(now())
  datumint   String?
}
```

Everything is updated with my postgresql db, and works if I remove datumint and dateint from everything. Yes its called int but is a string. If I remove datumint and try to add anything else it also gets an error. Its like im stuck with these variables. No idea what to do!

Any help appreciated! Also sorry for the bad formatting, im new to this site.

**Comment by austincrim** [comment]
Hey @tjugosex 👋,

What error messages are you getting?

**Reply by Figure26** [reply]
I get the 500 error in console from my api catch

**Reply by austincrim** [reply]
Does your `console.error(e)` log anything?

**Reply by Figure26** [reply]
Sorry I dont know how to check that, this code is pasted from someone else. I got it to work now by restarting the site, something Id never thought about before. Do I need to restart everytime I add new columns? Thanks anyway!

**Reply by austincrim** [reply]
That's great! Yep, if you make changes to your `schema.prisma`, you will need to re-run `prisma generate` and restart your app.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
